app/build.gradle

```diff
@@ -86,6 +87,7 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:26.1.0'
     implementation 'com.android.support:palette-v7:26.1.0'
     implementation 'com.android.support:cardview-v7:26.1.0'
+    implementation 'com.android.support:multidex:1.0.2'
```

Comment:
Why this line?

Reply:
Yes, it is not necessary.

I add it as for this:
multiDexEnabled true

Reply:
Remove and I'll pull

Reply:
I already remove and rebase it in c69742a.

Reply:
Kudos for rebasing